core/src/elements/ons-page.js

```diff
@@ -380,14 +363,16 @@ class PageElement extends BaseElement {
     this.setAttribute('_compiled', '');
   }
 
-  _registerExtraElement(element) {
-    let extra = util.findChild(this, '.page__extra');
-    if (!extra) {
-      extra = util.create('.page__extra', {zIndex: 10001});
-      this.appendChild(extra);
+  _elementShouldBeMoved(el) {
+    if (el.className.match(/\bpage__/)) {
```

Comment:
Maybe you can use `/\bpage__content/.test(...)` instead since we are not using the match for anything.